src/main/java/net/testudobank/TestudoBankRepository.java

```diff
@@ -102,6 +112,14 @@ public static void deleteRowFromOverdraftLogsTable(JdbcTemplate jdbcTemplate, St
     jdbcTemplate.update(deleteRowFromOverdraftLogsSql);
   }
 
+  public static void insertRowToTransferLogsTable(JdbcTemplate jdbcTemplate, String customerID, String recipientID, int transferAmount) { 
```

Comment:
This is a useful helper method that can be kept for the future.

Reply:
Cool. Won't remove this then.